src/GovUk.Education.ExploreEducationStatistics.Public.Data.Scripts/Commands/SeedDataCommand.cs

```diff
@@ -270,30 +267,31 @@ public async Task Generate()
             IList<MetaFileRow> metaFileRows,
             HashSet<string> allowedColumns)
         {
-            var totalResults = await _duckDb.QuerySingleAsync<int>($"SELECT COUNT(*) FROM '{_dataFilePath}'");
+            var totalResults = await _duckDbConnection.SqlBuilder(
+                $"""
+                 SELECT COUNT(*)
+                 FROM '{_dataFilePath:raw}'
+                 """
+            ).QuerySingleAsync<int>(cancellationToken: _cancellationToken);
 
-            var geographicLevels = (await _duckDb.QueryAsync<string>(
-                    new CommandDefinition(
-                        $"""
-                         SELECT DISTINCT geographic_level
-                         FROM read_csv_auto('{_dataFilePath}', ALL_VARCHAR = true)
-                         """,
-                        cancellationToken: _cancellationToken
-                    )
-                ))
+            var geographicLevels =
+                (await _duckDbConnection.SqlBuilder(
+                    $"""
+                     SELECT DISTINCT geographic_level
+                     FROM read_csv('{_dataFilePath:raw}', ALL_VARCHAR = true)
+                     """
+                ).QueryAsync<string>(cancellationToken: _cancellationToken))
                 .Select(EnumToEnumLabelConverter<GeographicLevel>.FromProvider)
                 .ToList();
 
-            var timePeriods = (await _duckDb.QueryAsync<(string TimePeriod, string TimeIdentifier)>(
-                    new CommandDefinition(
-                        $"""
-                         SELECT DISTINCT time_period, time_identifier
-                         FROM read_csv_auto('{_dataFilePath}', ALL_VARCHAR = true)
-                         ORDER BY time_period
-                         """,
-                        cancellationToken: _cancellationToken
-                    )
-                ))
+            var timePeriods = (await _duckDbConnection.SqlBuilder(
+                    $"""
+                     SELECT DISTINCT time_period, time_identifier
+                     FROM read_csv('{_dataFilePath:raw}', ALL_VARCHAR = true)
+                     ORDER BY time_period
+                     """
+                ).QueryAsync<(string TimePeriod, string TimeIdentifier)>(cancellationToken: _cancellationToken)
+                )
```

Comment:
Nitpick, but the parens on its own like this is a bit weird. Similar thing happens further down as well.